src/caches/basic_method_caches.jl

```diff
@@ -115,3 +115,111 @@ function alg_cache(alg::RKMilCommute,prob,u,ΔW,ΔZ,p,rate_prototype,noise_rate_
   Kj = zero(u); Dgj = zero(noise_rate_prototype)
   RKMilCommuteCache(u,uprev,du1,du2,K,gtmp,L,I,Dg,mil_correction,Kj,Dgj,tmp)
 end
+
+struct WangLi3SMil_AConstantCache <: StochasticDiffEqConstantCache end
```

Comment:
these should all get their own file.

Reply:
Will do. And If you could check whether I covered all the convergence tests?

Reply:
what you have should be good.

Reply:
I've moved these methods to a new file and I've added a few print statements because travis for rerporting timeouts.